frontend/src/pages/modelServing/screens/projects/ServingRuntimeDetails.tsx

```diff
@@ -47,7 +47,7 @@ const ServingRuntimeDetails: React.FC<ServingRuntimeDetailsProps> = ({ obj }) =>
       <DescriptionListGroup>
         <DescriptionListTerm>Accelerator</DescriptionListTerm>
         <DescriptionListDescription>
-          {accelerator.accelerator?.spec.displayName || 'unknown'}
+          {accelerator.accelerator?.spec.displayName || 'None'}
```

Comment:
Would this cause a problem with existing Model Servers?

Reply:
for exisitng model servers, the useAcceleratorState still gets called and we evaluate what accelerator they are using. In this flow exisitng serving runtimes would wither fall into the `useExisiting` state or the `Nvidia GPU` state. It wont be linked to a profile, but it would be uses the _fake_ profile (soft migration or whatever we call it)